coremltools/models/model.py

```diff
@@ -8,6 +8,7 @@
 import shutil as _shutil
 import tempfile as _tempfile
 import warnings as _warnings
+import numpy
```

Comment:
This import should be hidden. So do `import numpy as _numpy`. Then later in the file when you access numpy, do it with `_numpy.` rather than `numpy.`.